src/app/components/pages/TagsIndex.jsx

```diff
@@ -55,6 +55,8 @@ export default class TagsIndex extends React.Component {
         //console.log('-- TagsIndex.render -->', tagsAll.toJS());
         const { order } = this.state;
         let tags = tagsAll;
+        //filter out non-strings
+        tags = tags.filter(tag => typeof tag === 'string');
```

Comment:
this inline funciton is reused a few times -- even though it's simple (for now) it should be a named function